06-Tables/Advanced_Tables.ipynb

```diff
@@ -545,8 +527,8 @@
     "    sources['x0'] = np.round(sources['x_mean']).astype(int)\n",
     "    sources['y0'] = np.round(sources['y_mean']).astype(int)\n",
     "    \n",
-    "    # Generate list of cutout images around each source\n",
-    "    img = photutils.datasets.make_gaussian_sources(image_shape=(200, 200), source_table=out)\n",
+    "    # Generate list of cutout images around each source (syntax fixed)\n",
```

Comment:
Remove _(syntax fixed)_ comment

Reply:
Sorry, I thought I caught all of those comments about syntax, this is not an issue to edit.